kubos/utils/git_utils.py

```diff
@@ -32,23 +32,45 @@ def get_tag_list(repo):
     tags = repo.tags
     tag_list = []
     for tag in tags:
-        tag_list.append(tag)
+        tag_list.insert(0, tag.name)
     return tag_list
 
 
-def print_tag_list(tag_list):
+def filter_minor_tags(display_num, tag_list):
```

Comment:
I know I was a little sloppy in defining "minor releases" as releases that were triggered from CI/CD with the minor version number still being a specific field in the version as well. I think a better term for these releases could be "nightly", "alpha", "CD" or something along those lines.. 

The version format you suggest would be nice for consistency across our practices if we decided to start using that format for our modules. 

But one thing to keep in mind is that the tag/release number that is pushed to the kubos repo is never used or touched by yotta. The `kubos use` command simply checks out the git tag and then starts the linking process.